### PR TITLE
Site refactor

### PR DESCRIPTION
**Pull request by mirons-google**
This reorganizes the site into the newly decided categories.

### CODE REVIEW

**Comment by mirons-google** [comment]
Please ignore the github.com links being broken.

File: _docs/configure/enforcer/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ order: 201
 ---
 # {{ page.title }}
 
-This quickstart describes how to get started with Forseti Enforcer. Forseti
+This document describes how to get started with Forseti Enforcer. Forseti
```
Comment (Tabrarian) [inline _docs/configure/enforcer/index.md]: Nit for simplicity: "This page describes..."

File: _docs/configure/explain/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ order: 202
 IAM Explain is not part of the main Forseti releases from master. Please
 deploy from the [IAM Explain experimental branch](https://github.com/GoogleCloudPlatform/forseti-security/tree/explain-experimental).
 
-This quickstart describes how to set up IAM Explain for Forseti Security.
+This document describes how to set up IAM Explain for Forseti Security.
```
Comment (Tabrarian) [inline _docs/configure/explain/index.md]: page

File: _docs/configure/inventory/index.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ order: 002
 ---
 # {{ page.title }}
 
-This quickstart describes how to get started with Forseti Inventory. Forseti
+This document describes how to get started with Forseti Inventory. Forseti
```
Comment (Tabrarian) [inline _docs/configure/inventory/index.md]: page

File: _docs/configure/scanner/index.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ order: 101
 ---
 # {{ page.title }}
 
-This quickstart describes how to get started with Forseti Scanner. Forseti
+This document describes how to get started with Forseti Scanner. Forseti
```
Comment (Tabrarian) [inline _docs/configure/scanner/index.md]: page

File: _docs/setup/gcp-deployment.md [hunk 5]
```diff
@@ -145,8 +146,8 @@ At this point, Forseti should be installing on the GCE instance that was
 created from the Deployment Manager script. You can ssh into the GCE instance 
 and watch the /tmp/deployment.log to watch the progress.
 
-To ssh to your GCE instance, you can either use `gcloud compute ssh` ([official docs](https://cloud.google.com/sdk/gcloud/reference/compute/ssh)) or go to the GCE 
-instance details page from Google Cloud Console > Compute Engine, then click the 
+To ssh to your GCE instance, you can either use `gcloud compute ssh` ([official docs](https://cloud.google.com/sdk/gcloud/reference/compute/ssh))
```
Comment (Tabrarian) [inline _docs/setup/gcp-deployment.md]: Compute Engine

(Google branding guidelines)

File: _docs/setup/gcp-deployment.md [hunk 6]
```diff
@@ -145,8 +146,8 @@ At this point, Forseti should be installing on the GCE instance that was
...
+or go to the GCE instance details page from Google Cloud Console > Compute Engine, then click the
```
Comment (Tabrarian) [inline _docs/setup/gcp-deployment.md]: Compute Engine

File: _docs/setup/gcp-deployment.md [hunk 7]
```diff
@@ -145,8 +146,8 @@ At this point, Forseti should be installing on the GCE instance that was
 created from the Deployment Manager script. You can ssh into the GCE instance
```
Comment (Tabrarian) [inline _docs/setup/gcp-deployment.md]: Compute Engine

File: _docs/setup/gcp-deployment.md [hunk 8]
```diff
@@ -118,12 +118,13 @@ other deployments):
...
+* **output_path**: Set this to gs://YOUR_SCANNER_BUCKET/scanner_violations
+(where YOUR_SCANNER_BUCKET is the `SCANNER_BUCKET` value you used in deploy-forseti.yaml).
 * **rules_path**: Set this to "/home/ubuntu/forseti-security/rules"
 
 ### Customize Forseti Rules
 
-Customize your Forseti rules by following [this guide]({% link _docs/quickstarts/scanner/rules.md %}).
+Customize your Forseti rules by following [this guide]({% link _docs/configure/scanner/rules.md %}).
 
 ### Move Configuration to GCS
```
Comment (Tabrarian) [inline _docs/setup/gcp-deployment.md]: Cloud Storage

(Google branding guidelines)

**Comment by mirons-google** [comment]
All requested changes were made. Thanks!

**Pull request merged**